packages/trader/src/exchange-adapters/CreateExchangeAdapter.ts

```diff
@@ -2,7 +2,8 @@ import winston from "winston";
 import Web3 from "web3";
 
 const assert = require("assert");
-const { UniswapTrader } = require("./UniswapTrader");
+const { UniswapV2Trader } = require("./UniswapV2Trader");
+const { UniswapV3Trader } = require("./UniswapV3Trader");
 
 async function createExchangeAdapter(
```

Comment:
Might be useful to define a return type for this function (`Promise<ExchangeAdapterInterface | null>` should probably work?) so it doesn't default to any.

Reply:
good idea! done